src/vivarium_testing_utils/automated_validation/visualization/dataframe_utils.py

```diff
@@ -110,7 +107,14 @@ def format_draws_sample(draw_list: list[int], source: DataSource) -> str:
     draw_list = sorted(draw_list)
     if source == DataSource.SIM:
         # Display all draws run in the simulation
-        return str(draw_list)
+        if len(draw_list) < 6:
```

Comment:
I don't think you need this if branch. Your `else` block should be able to deal with a draw list with length less than 6.